V2/server/controllers/articlecontroller.js

```diff
@@ -0,0 +1,48 @@
+
+import dotenv from 'dotenv';
+import Model from '../model/dbqueries';
+
+import verifytoken from '../../../V1/server/helpers/tokens';
+
+
+let authorid;
+dotenv.config();
+
+class articleController {
+  static model() {
+    return new Model('articles');
+  }
+
+  static createArticle = async (req, res) => {
+    const token = req.header('token');
+    const decode = verifytoken.verifyToken(token);
+    try {
+      let {
+        title,
+        article,
+        category,
+
+      } = req.body;
+      authorid = decode.userEmail;
```

Comment:
Its because I will be using it in many functions in my article controller every time to get author id, I mean like for edit article I will also need to take the user's email and  use it again to check if the user email that wants to edit ana article is the one that created it, that's why I declared it as global but for now I can change it to const no problem